Text/HealthcareTA/HealthcareTA.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using AzureCognitiveSearch.PowerSkills.Common;
+using Azure.AI.TextAnalytics;
+using Azure;
+
+namespace AzureCognitiveSearch.PowerSkills.Text.HealthcareTA
+{
+    public static class HealthcareTA
+    {
+        private static readonly string healthcareApiEnvEndpoint = "HEALTHCARE_API_ENDPOINT";
+        private static readonly string healthcareApiEnvKey = "HEALTHCARE_API_KEY";
+        private static readonly int defaultTimeout = 230;
+        private static readonly int maxCharLength = 5000;
+
+        [FunctionName("HealthcareTA")]
+        public static async Task<IActionResult> Run(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = null)] HttpRequest req,
+            ILogger log,
+            ExecutionContext executionContext)
+        {
+            string skillName = executionContext.FunctionName;
+            IEnumerable<WebApiRequestRecord> requestRecords = WebApiSkillHelpers.GetRequestRecords(req);
+            if (requestRecords == null)
+            {
+                return new BadRequestObjectResult($"{skillName} - Invalid request record array.");
+            }
+
+            // Get Endpoint and access key from App Settings
+            string apiKey = Environment.GetEnvironmentVariable(healthcareApiEnvKey, EnvironmentVariableTarget.Process);
+            string apiEndpoint = Environment.GetEnvironmentVariable(healthcareApiEnvEndpoint, EnvironmentVariableTarget.Process);
+            if (apiKey == null || apiEndpoint == null)
+            {
+                return new BadRequestObjectResult($"{skillName} - Healthcare Text Analytics API key or endpoint is missing. Make sure to set them in the Environment Variables.");
+            }
+
+            var client = new TextAnalyticsClient(new Uri(apiEndpoint), new AzureKeyCredential(apiKey));
+
+            // Get a custom timeout from the header, if it exists. If not use the default timeout.
+            int timeout;
+            if (!int.TryParse(req.Headers["SkillTimeout"].ToString(), out timeout))
```

Comment:
Make sure you make a note when you document this that it needs to be in seconds.

Reply:
Also, we should validate to make sure its between 0 and 230 (your default) since anything else wouldn't work